utils/mock-data-loader/insert-mocks-gef.js

```diff
@@ -22,18 +22,20 @@ const insertMocks = async (token) => {
   const makerUserId = allUsers.find((user) => user.username === BANK1_MAKER1.username)._id;
   const latestEligibilityCriteria = await api.latestEligibilityCriteria(token);
 
+  const makerToken = await portalApi.loginViaPortal(BANK1_MAKER1);
+
   // eslint-disable-next-line no-unused-vars
```

Comment:
Can this please be removed, appreciate this is not added in your PR. Thanks

Reply:
And remaining below eslint rules please.

Reply:
I removed disabling for eslint rules without much of code changes. Interesting if it works.

It would be nice to avoid modifying code as it is quite old and works.